Fixes #77 - Java 8 compatibility.

sbordet opened the pull request:
Updated dependencies for ALPN.
Updated usage of Java 11 API back to Java 8 APIs.

Signed-off-by: Simone Bordet <simone.bordet@gmail.com>

sbordet committed:
Fixes #77 - Java 8 compatibility.

Updated dependencies for ALPN.
Updated usage of Java 11 API back to Java 8 APIs.

Signed-off-by: Simone Bordet <simone.bordet@gmail.com>

joakime commented:
So this version will have both of the OpenJDK8 and Java9 versions of the ALPNProcessor libs in it's uber-jar?

sbordet committed:
Updated README.
Added both ALPN implementations to uber jar, so that it can be used with any Java version.

Signed-off-by: Simone Bordet <simone.bordet@gmail.com>

sbordet commented:
@joakime yes.
